NDB.Covid19/NDB.Covid19/Locales/nn.json

```diff
@@ -317,6 +317,7 @@
   "SETTINGS_GENERAL_MORE_INFO_LINK": "https://www.fhi.no/om/smittestopp/tekster-pa-andre-sprak",
   "SETTINGS_GENERAL_MORE_INFO_BUTTON_TEXT": "Les om Smittestopp på andre språk på: www.fhi.no/om/smittestopp/tekster-pa-andre-sprak.",
   "SETTINGS_GENERAL_ACCESSIBILITY_MORE_INFO_BUTTON_TEXT": "Les om Smittestopp på andre språk på fhi.no/om/smittestopp/tekster-pa-andre-sprak",
+  "DAILY_NUMBERS_ACCESSIBILITY_BACK_BUTTON": "Tilbake",
```

Comment:
What about other languages?

Reply:
We are missing translations for all texts in other languages on this page. It's added to the sheet, so it will be added when we do the big text update

Reply:
You can add the text for all languages using value for SETTINGS_CHILD_PAGE_ACCESSIBILITY_BACK_BUTTON key.

Reply:
Done